mhapsite/mhap/models.py

```diff
@@ -15,6 +15,11 @@
 import requests, json, os
 MSFT_COGSERV_KEY = os.environ.get('MSFT_COGSERV_KEY')
 
+# for keyword searches for important topics
+import re
+depression_patterns = re.compile(r'(depression)|(depressed)|(hopeless)|(worthless)', flags=re.I)
+suicide_patterns = re.compile(r'(suicid)|(kill me)|(kill myself)|(I was dead)', flags=re.I)
```

Comment:
Mark this looks good for now. We could further encapsulate it in another python file and import that too.